plugins/connection/persistent.py

```diff
@@ -72,6 +72,9 @@ def run(self):
             "starting connection from persistent connection plugin",
             host=self._play_context.remote_addr,
         )
+        display.warning(
+            "connection local support from netcommon is deprecated and will be removed at 01-01-2027"
```

Comment:
```suggestion
            "support for connection local has been deprecated from ansible.netcommon and will be removed in a release after 01-01-2027"
```

Reply:
Also, will it be possible to use the `deprecated()` method instead of warning.

https://github.com/ansible/ansible/blob/devel/lib/ansible/utils/display.py#L573